### PR TITLE
Fixes #19979: Cover all supported server OS in the PR automated tests.

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19979

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: Jenkinsfile [hunk 1]
```diff
@@ -3,18 +3,48 @@ import org.gradiant.jenkins.slack.SlackNotifier
...
+    stage('ncf-tests pull-request') {
+        when { changeRequest() }
+        deleteDir()
+        dir('ncf') {
+            checkout scm
+            sh script: 'git log -1 --pretty=%B'
+        }
+        script {
+            String[] versions = [
+                "ci/rudder-6.1-nightly",
```
Comment (amousset) [inline Jenkinsfile:33]: it is the agent version for the tests?

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**